packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -163,27 +163,16 @@ const WorkspaceHeader: React.FC = () => {
               <Icon icon="logo" size="medium" />
             </Link>
           </div>
-          <Tooltip
```

Comment:
@Sheersendu the fixed looks good, but there are unrelated changes. Can you please checkout from branch next and create a new PR?

Reply:
Hi @abrl91, this change is related to another issue that I was working on. Is it necessary to create e new PR, if I revert back the changes for this file and push will be fine then?

Reply:
@abrl91 required changes have been made, hope its fine now :)